### ISSUE TITLE
Spanish translation progress

### ISSUE THREAD

**Issue by fagossa** [thecodix/book#5]
The current state of the _spanish_ translation of the second edition is as follows:
 
- [ ] SUMMARY.md
- [ ] appendix-00.md
- [ ] appendix-01-keywords.md
- [ ] appendix-02-operators.md
- [ ] appendix-03-derivable-traits.md
- [ ] appendix-04-macros.md
- [x] appendix-05-translation.md (**@osovich**)
- [ ] appendix-06-newest-features.md
- [x] ch01-00-introduction.md (**PR6 @fagossa**)
- [x] ch01-01-installation.md (**PR7 @jdamu**)
- [x] ch01-02-hello-world.md (**@osovich**)
- [x] ch01-03-how-rust-is-made-and-nightly-rust.md (**@osovich**)
- [x] ch02-00-guessing-game-tutorial.md (**@thecodix**)
- [x] ch03-00-common-programming-concepts.md (**@chicooou**)
- [x] ch03-01-variables-and-mutability.md (**@chicooou**)
- [x] ch03-02-data-types.md
- [x] ch03-03-how-functions-work.md (**@chicooou**)
- [ ] ch03-04-comments.md
- [x] ch03-05-control-flow.md
- [ ] ch04-00-understanding-ownership.md
- [x] ch04-01-what-is-ownership.md (**@chicooou**)
- [x] ch04-02-references-and-borrowing.md (**@chicooou**)
- [x] ch04-03-slices.md
- [x] ch05-00-structs.md (**@chicooou**)
- [ ] ch05-01-defining-structs.md
- [x] ch05-02-example-structs.md
- [ ] ch05-03-method-syntax.md
- [ ] ch06-00-enums.md
- [ ] ch06-01-defining-an-enum.md
- [ ] ch06-02-match.md
- [ ] ch06-03-if-let.md
- [x] ch07-00-modules.md (**@chicooou**)
- [x] ch07-01-mod-and-the-filesystem.md (**@chicooou**)
- [x] ch07-02-controlling-visibility-with-pub.md (**@chicooou**)
- [ ] ch07-03-importing-names-with-use.md
- [x] ch08-00-common-collections.md (**@siersod-utopian**)
- [x] ch08-01-vectors.md
- [x] ch08-02-strings.md (**@jossue33**)
- [ ] ch08-03-hash-maps.md
- [x] ch09-00-error-handling.md (**@siersod-utopian**)
- [ ] ch09-01-unrecoverable-errors-with-panic.md
- [ ] ch09-02-recoverable-errors-with-result.md
- [ ] ch09-03-to-panic-or-not-to-panic.md
- [x] ch10-00-generics.md (**@jossue33**)
- [x] ch10-01-syntax.md (**@jossue33**)
- [ ] ch10-02-traits.md
- [ ] ch10-03-lifetime-syntax.md
- [ ] ch11-00-testing.md
- [ ] ch11-01-writing-tests.md
- [ ] ch11-02-running-tests.md
- [ ] ch11-03-test-organization.md
- [ ] ch12-00-an-io-project.md
- [ ] ch12-01-accepting-command-line-arguments.md
- [ ] ch12-02-reading-a-file.md
- [ ] ch12-03-improving-error-handling-and-modularity.md
- [ ] ch12-04-testing-the-librarys-functionality.md
- [ ] ch12-05-working-with-environment-variables.md
- [ ] ch12-06-writing-to-stderr-instead-of-stdout.md
- [ ] ch13-00-functional-features.md
- [ ] ch13-01-closures.md
- [ ] ch13-02-iterators.md
- [ ] ch13-03-improving-our-io-project.md
- [ ] ch13-04-performance.md
- [ ] ch14-00-more-about-cargo.md
- [ ] ch14-01-release-profiles.md
- [ ] ch14-02-publishing-to-crates-io.md
- [ ] ch14-03-cargo-workspaces.md
- [ ] ch14-04-installing-binaries.md
- [ ] ch14-05-extending-cargo.md
- [ ] ch15-00-smart-pointers.md
- [ ] ch15-01-box.md
- [ ] ch15-02-deref.md
- [ ] ch15-03-drop.md
- [ ] ch15-04-rc.md
- [ ] ch15-05-interior-mutability.md
- [ ] ch15-06-reference-cycles.md
- [ ] ch16-00-concurrency.md
- [ ] ch16-01-threads.md
- [ ] ch16-02-message-passing.md
- [ ] ch16-03-shared-state.md
- [ ] ch16-04-extensible-concurrency-sync-and-send.md
- [ ] ch17-00-oop.md
- [ ] ch17-01-what-is-oo.md
- [ ] ch17-02-trait-objects.md
- [ ] ch17-03-oo-design-patterns.md
- [x] ch18-00-patterns.md (**@chicooou**)
- [ ] ch18-01-all-the-places-for-patterns.md
- [ ] ch18-02-refutability.md
- [ ] ch18-03-pattern-syntax.md
- [x] ch19-00-advanced-features.md (**@Santu41**)
- [x] ch19-01-unsafe-rust.md (**@Santu41**)
- [ ] ch19-02-advanced-lifetimes.md
- [ ] ch19-03-advanced-traits.md
- [ ] ch19-04-advanced-types.md
- [ ] ch19-05-advanced-functions-and-closures.md
- [ ] ch20-00-final-project-a-web-server.md
- [ ] ch20-01-single-threaded.md
- [ ] ch20-02-slow-requests.md
- [ ] ch20-03-designing-the-interface.md
- [ ] ch20-04-storing-threads.md
- [ ] ch20-05-sending-requests-via-channels.md
- [ ] ch20-06-graceful-shutdown-and-cleanup.md

**Comment by jdamu** [comment]
Hi!

I've done a pull request with ch01-01-installation.md translation to @thecodix repo

**Comment by thecodix** [comment]
Requests have been merged, thanks a lot!!

**Comment by chicooou** [comment]
Hello, I would like to translate to Spanish but I do not know how. I would like to do it by means of pull request :)

**Comment by thecodix** [comment]
Yes please, open a pull request with the suggested changes and we will merge it to master, the list of chapters that are pending is still huge

**Comment by dbucci24** [comment]
@thecodix Hello! I would like to collaborate with the translation! I can do it? 

Regards! :)

**Comment by thecodix** [comment]
hey @dbucci24 definitely, please take a look at the remaining items, there is still a lot to do, but the translation is been active in the last week thanks to @chicooou specially

Please make sure there is not a PR already open for a chapter you want to do :)

Any questions feel free to ask

**Comment by thecodix** [comment]
Btw @chicooou will have to do sooner or later a merge with the main branch of the english translation, small details (or late chapters) may have changed, I can handle small changes but will be good if we dont translate twice a whole chapter :)

Will try to do it asap

**Comment by dbucci24** [comment]
@thecodix thank you! I will work hard in translation! 😃

**Comment by chicooou** [comment]
@thecodix  Should I do it? How do I do it? :)

**Comment by thecodix** [comment]
Hey everyone, just merged the master from rust-lang/book into my repository, I did a huge effort to try and merge most possible, but would kindly ask you to review each chapter you translated individually since there are minor additions or modifications in a few of them

Thanks again!

**Comment by thecodix** [comment]
Also please consider coming to this reference page, and updating it is possible :) for words that you are unsure how to translate, to keep same conventions for the whole book

https://github.com/thecodix/book/wiki/Translation-of-complicated-words

**Comment by mateovelilla** [comment]
Hi, Is this repo official for the Spanish translation?
